File: api/v2010/member/read-example-1/read-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

members = @client.queues('QU5ef8732a3c49700934481addd5ce1659').members.list

members.each do |record|
  puts record.call_sid
end

File: api/v2010/media/read-example-1/read-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account.Message;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var media = MediaResource.Read(
            pathMessageSid: "MM800f449d0399ed014aae2bcc0cc2f2ec"
        );

        foreach(var record in media)
        {
           Console.WriteLine(record.Sid);
        }
    }
}
